crown-court-contribution/build.gradle

```diff
@@ -38,8 +38,23 @@ dependencies {
 	implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web'
 	implementation 'org.postgresql:postgresql'
 	implementation group: 'org.apache.commons', name: 'commons-lang3', version: versions.commonsLang3Version
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+	implementation group: 'org.springframework.boot', name: 'spring-boot-starter'
+	implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web'
```

Comment:
Line 43 is a duplicate of line 38. Can you please review all the dependencies and formatting? Thanks